kumascript/macros/DiscussionList.ejs

```diff
@@ -160,10 +160,7 @@ listElem += ">"; //Final ">" of the open <ul> tag
 
 
 function makeListTarget(param) {
-    function startsWith (string, prefix) {
-        return string.slice(0, prefix.length) == prefix;
-    }
-    if (startsWith(param, 'https://') || startsWith(param, 'http://')) {
+    if (param && (param.startsWith('https://') || param.startsWith('http://'))) {
```

Comment:
Check for `param` being truthy might be unnecessary, I added it just for peace of mind.

Reply:
That's a scope issue but You can add a default to the function.
No need to do It now.